queries.sql

```diff
@@ -1,3 +1,10 @@
 /*Queries that provide answers to the questions from all projects.*/
 
-SELECT * from animals WHERE name = 'Luna';
+SELECT * FROM animals WHERE name like '%mon';
+SELECT name FROM animals WHERE date_of_birth >= '2016-01-01' AND date_of_birth <= '2019-01-01';
+SELECT name FROM animals WHERE neutered = 't' AND escape_attempts < 3;
+SELECT name FROM animals where name LIKE '%Agumon' OR name LIKE '%Pikachu';
```

Comment:
- [ ] Here the question is to list the Date of birth of all animals named either "Agumon" or "Pikachu", you should select the date of birth from animals 😉 😉